chromium/README-Linux.md

```diff
@@ -20,26 +20,26 @@ gclient sync
 cd $CHROMIUM_ROOT/third_party/boringssl/src
 git remote add oqs-bssl https://github.com/open-quantum-safe/boringssl
 git fetch oqs-bssl
-git checkout -b oqs-bssl-master 1ca41b49e9198f510991fb4f350b4a5fd4c1d5ff
+git checkout -b oqs-bssl-master c0a0bb4d1243952819b983129c546f9ae1c03008
 ```
 
 ### 4. Clone and Build liboqs
 
 Choose a directory to store the liboqs source code and use the `cd` command to move to that directory. We will use ninja to build liboqs.
 
 ```shellscript
-git clone https://github.com/open-quantum-safe/liboqs.git --branch 0.8.0 --single-branch
+git clone --branch main https://github.com/open-quantum-safe/liboqs.git && git checkout 890a6aa448598a019e72b5431d8ba8e0a5dbcc85
```

Comment:
Isn't this a contradiction in itself? Cloning "main" but then checking out a specific commit?